### PR TITLE
FACES-3050 Not all TCK test runner shell scripts have commands that i…

### PR DESCRIPTION
**Pull request by pcwhite**
…nvoke the new test framework

### CODE REVIEW

**Comment by pcwhite** [comment]
@ngriffin7a can be backported to 3.x and 2.x!

File: tck/bridge-tck-main-portlet/test-liferay62-glassfish.sh [hunk 1]
```diff
@@ -1,2 +1,2 @@
 #!/bin/sh
-mvn surefire-report:report -P tck,liferay,liferay62,glassfish
+mvn verify -P selenium,liferay62,chrome,glassfish "$@"
```
Comment (stiemannkj1) [inline tck/bridge-tck-main-portlet/test-liferay62-glassfish.sh:2]: @pcwhite, is the `chrome` profile necessary? I think it's the default.

Comment (pcwhite) [reply tck/bridge-tck-main-portlet/test-liferay62-glassfish.sh:2]: @stiemannkj1 no, it's not necessary.  But, it's consistent with the other `liferay70` scripts. Also, is `chrome` always going to be the default browser? If so, I can remove the `chrome` profile for all the scripts.

**Comment by ngriffin7a** [comment]
Merged and backported. Thank you.

**Pull request closed without merging**